lib/ofac/index.js

```diff
@@ -42,13 +42,13 @@ function makeCompatible (nameParts) {
 
 function match (nameParts, birthDateString, options) {
   const {debug} = options
+
   if (!structs) {
     const message = 'The OFAC data sources have not been loaded yet.'
-    return Promise.reject(new Error(message))
+    throw new Error(message)
```

Comment:
I think it's better to raise here because before the fix of the `config.sanctionsVerificationActive`, it was behaving in a wrong way.

Reply:
the rejected promise where passed [here](https://github.com/lamassu/lamassu-server/pull/137/files#diff-6fdd173c94cfbb369224848953c3921eR57) as results variable and then was processed here https://github.com/lamassu/lamassu-server/pull/137/files#diff-6fdd173c94cfbb369224848953c3921eR21 somehow.
BTW it wasn't catched outside as an Error

Reply:
Yeah good point, but I think we should catch errors in `validateOfac`, return `false`, and log the error using `logger.error()`.